_includes/macros.html

```diff
@@ -166,11 +166,11 @@ <h1 class="h5 {{'u-visually-hidden' if hide_heading}}">{{ text }}</h1>
    ========================================================================== #}
 
 {% macro format_address(contact) %}
-    {{ contact.street if contact.street }}<br>
-    {{ contact.street_2 + '<br>'|safe if contact.street_2 }}
-    {{ contact.city + ',' if contact.city }}
-    {{ (contact.state + '&nbsp;')|safe if contact.state }}
-    {{ contact.zip_code if contact.zip_code }}
+    {{ contact.street if contact.street else '' }}<br>
+    {{ contact.street_2 + '<br>'|safe if contact.street_2 else '' }}
+    {{ contact.city + ',' if contact.city else '' }}
+    {{ (contact.state + '&nbsp;')|safe if contact.state else '' }}
+    {{ contact.zip_code if contact.zip_code else '' }}
```

Comment:
It was the first solution I landed on that silenced this warning:

https://github.com/mitsuhiko/jinja2/blob/46acbf02ed9ab58c7a92553c95790e75bdc3c930/jinja2/compiler.py#L1600

The result of an inline if, that is false and missing an else, is "Undefined". 

There might be other ways around it, but I figured it might be a good place to practice "explicit is better than implicit".

http://jinja.pocoo.org/docs/dev/templates/#if-expression